.github/workflows/export_category.yml

```diff
@@ -60,8 +60,8 @@ jobs:
 
       - name: export
         run: |
+          mkdir -p .staging/${{matrix.category}}
```

Comment:
Not sure if it would work if it comes out. Because of how the dump metadata works, the path has to exists before trying to output so I resorted to this. Alternative design would be doing this with Python in the metadata dump function.

Reply:
Interesting! Good catch. In that case yea I think we should create the `.staging/<category>` folder in `dump_metadata`. Just copy/paste the two lines from the `collate` function

Reply:
This issue still stands I think